src/Microsoft.AspNetCore.BuildTools.ApiCheck/Loader/AssemblyLoader.cs

```diff
@@ -1,66 +1,33 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System;
-using System.Collections.Generic;
-using System.IO;
 using System.Linq;
 using System.Reflection;
-using System.Runtime.InteropServices;
-using Microsoft.DotNet.InternalAbstractions;
-using Microsoft.DotNet.ProjectModel;
-using Microsoft.DotNet.ProjectModel.Compilation;
-using Microsoft.DotNet.ProjectModel.Graph;
-using Microsoft.Extensions.DependencyModel;
+#if NETCOREAPP1_1
 using NuGet.Frameworks;
+using NuGet.ProjectModel;
+#endif
+using NugetReferenceResolver;
+using System.IO;
 
 namespace ApiCheck
 {
     public abstract class AssemblyLoader
     {
         public static Assembly LoadAssembly(
                 string assemblyPath,
-                string projectJson,
-                string configuration,
-                string packagesFolder)
+                string assetsJson)
         {
-            var ctx = CreateProjectContext(projectJson, packagesFolder);
-            var exporter = ctx.CreateExporter(configuration);
-            var dependencies = exporter.GetDependencies().ToArray();
             
-#if NETCOREAPP1_0
-            var loader = new CoreClrAssemblyLoader(dependencies, ctx.RuntimeIdentifier, ctx.GetOutputPaths(configuration)?.CompilationOutputPath);
+#if NETCOREAPP1_1
+            var lockFile = new LockFileFormat().Read(assetsJson);
+            var graph = PackageGraph.Create(lockFile, "netcoreapp1.1");
```

Comment:
Can you make the target TFM a parameter instead of hard-coding it? This may vary from project to project.